src/lib/Sendbird.tsx

```diff
@@ -121,7 +119,7 @@ function Sendbird(props: SendbirdProviderProps) {
              */
             replyType: replyType == null
               ? initialConfig.groupChannel.channel.replyType
-              : replyType.toLowerCase() as GroupChannelConfig['channel']['replyType'],
+              : replyType.toLowerCase() as SBUConfig['groupChannel']['channel']['replyType'],
```

Comment:
Is there a safer alternative to type assertion for safer casting? If the `replyType` in React UIKit is refactored into a different form, it seems that there would be no way to detect it using tools like TypeScript.

Reply:
How about this way ? 
https://github.com/sendbird/sendbird-uikit-react/pull/614/commits/ef8f1c068361e1da36df5bf018a85c8ba6b28994